### PR TITLE
fix:issue #644

### LINKED ISSUE #644: 用户详情页面和修改密码逻辑错误
**Describe the bug**

创建了一个用户以后，不管使用任何账户，登录以后修改密码都是改的新创建的用户。

**To Reproduce**

创建一个用户

方式一：使用任意账户登录，点修改密码，都是改的相同一个账户的密码。
方式二：从权限控制页面进入用户管理，点任意用户名进入详情页，详情页显示的全部都是同一个一户。

**Environment**

使用的docker最新版安装

```
docker run -d -p 15010:15010 -p 8080:8080 -p 8090:8090 -p 8091:8091 -p 8093:8093 -p 8761:8761 -p 9000:9000 -p 9090:9090  polarismesh/polaris-server-standalone:latest
```

### PR DESCRIPTION
**Pull request by chuntaojun**
**Please provide issue(s) of this PR:**
Fixes #644 

**To help us figure out who should review this PR, please put an X in all the areas that this PR affects.**

- [ ] Configuration
- [ ] Docs
- [x] Installation
- [ ] Performance and Scalability
- [x] Naming
- [ ] HealthCheck
- [ ] Test and Release

**Please check any characteristics that apply to this pull request.**

- [ ] Does not have any user-facing changes. This may include API changes, behavior changes, performance improvements, etc.

### CODE REVIEW

File: auth/defaultauth/user.go [hunk 1]
```diff
@@ -587,6 +587,14 @@ func checkUpdateUser(req *api.User) *api.Response {
 		return api.NewUserResponse(api.BadRequest, req)
 	}
 
+	if err := checkMobile(req.Mobile); err != nil {
+		return api.NewUserResponse(api.InvalidUserMobile, req)
+	}
+
+    if err := checkEmail(req.Email); err != nil {
```
Comment (onecer) [inline auth/defaultauth/user.go:594]: 这个格式是不是要调整下

File: auth/defaultauth/utils.go [hunk 2]
```diff
@@ -140,7 +139,7 @@ func checkMobile(mobile *wrappers.StringValue) error {
 		return nil
 	}
 
-	if utf8.RuneCountInString(mobile.GetValue()) != 11 {
+	if utf8.RuneCountInString(mobile.GetValue()) != utils.MobilePhoneLength {
 		return errors.New("invalid mobile")
```
Comment (andrewshan) [inline auth/defaultauth/utils.go]: 这里写上什么原因校验不通过吧，比如当前字符串是啥，违反了多少字符数的限制

File: auth/defaultauth/utils.go [hunk 3]
```diff
@@ -140,8 +139,8 @@ func checkMobile(mobile *wrappers.StringValue) error {
 		return nil
 	}
 
-	if utf8.RuneCountInString(mobile.GetValue()) != 11 {
-		return errors.New("invalid mobile")
+	if utf8.RuneCountInString(mobile.GetValue()) != utils.MobilePhoneLength {
+		return errors.New("invalid mobile phone")
```
Comment (andrewshan) [inline auth/defaultauth/utils.go]: invalid mobile, current is %s, length must be less than %d

**Review by andrewshan — APPROVED** [review]
LGTM

**Pull request merged**